server/graphql/v1/mutations/orders.js

```diff
@@ -195,6 +195,9 @@ export async function createOrder(order, loaders, remoteUser, reqIp) {
     if (!collective) {
       throw new Error(`No collective found: ${order.collective.id || order.collective.website}`);
     }
+    if (collective.hasCustomContributionsDisabled) {
```

Comment:
This will refuse all contributions. When `hasCustomContributionsDisabled` is true, we must only refuse the ones without a tier.

Reply:
how to check if a collective doesn't have a tier ?

Reply:
will `!collective.tiers` work ?

Reply:
The goal of `hasCustomContributionsDisabled` is to prevent financial contributions (orders) that are sent without tier. So what we want is to make sure that `order.tier` has been set.